lib/components/NameForm.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+import Form from './Form';
+import TextInput from './TextInput';
+
+/**
+ * Class representing a generic Name Form
+ *
+ * This component returns a <form> with legend and some default Name fields: title, first/middle/last name, suffix
+ *
+ * Required props:
+ * - legend: string. sets the form legend
+ * - input: node. Series of field control types
+ *
+ * Optional props:
+ * - id: string. Sets the form's id attribute
+ * - inputClass: string. Adds any custom classes to TextInput fields
+ */
+export default function NameForm ({legend, input}) {
+  return (
+    <Form
+    legend="Name"
+    >
+      <TextInput
```

Comment:
Per my comment earlier, I would recommend that we put the fieldset and legend inside this component, not Form.  I'm also thinking that we might want to just return a fieldset as the top level rather than a form so that we can mix these fields in to a pre-existing form.  In that case we'd probably want to name this `NameFields` or something like that.  Thoughts?  @mstanaland thoughts?